bindings/python/Makefile

```diff
@@ -1,10 +1,11 @@
 default: build
 
-
-edlib: ../../edlib
-	cp -R ../../edlib .
+edlib:
+	# copy edlib from above if not already present
+	[ -d "edlib" ] && echo "Directory 'edlib' exists." || cp -R ../../edlib .
```

Comment:
Actually, we might want to copy it even if it already exists -> files might be old and we might want to overwrite them!

Reply:
Hmm, this might have been my logic behind setting it as `.PHONY`

Reply:
Changed to by phony and grab a fresh copy.